compiler/lsp-tests/src/Daml/Lsp/Test/Util.hs

```diff
@@ -0,0 +1,62 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE OverloadedStrings #-}
+module Daml.Lsp.Test.Util
```

Comment:
Why not Daml.LSP, with capital LSP, like the upstream module names.

Reply:
I first went with `DAML.LSP` which looked somewhat ugly and using `Daml.LSP` seemed somewhat inconsistent (side note: It looks like we are already inconsistent, we use `DAML` in the assistant modules but `DA.Daml` in other places) so I went with `Daml.Lsp` which seemed like the most consistent option. Upstream does seem to consistently use `LSP` in module names but for other identifiers they aren’t consistent either, e.g., there is `LSPVersion` and `LspFuncs` so all options are inconsistent in some way.

I don’t really care about the module name here so feel free to change it :slightly_smiling_face:

Reply:
It is indeed a bomb shell. I'm sure someone will unify everything one day.